.github/workflows/dependency-changelog.yml

```diff
@@ -32,7 +32,7 @@ permissions:
 
 jobs:
   dependency_changelog:
-    if: ${{ github.event.label.name == 'dependencies' }}
```

Comment:
In theory this would also kick-off if another label was added after the `dependencies` label, but nothing would happen since the changelog file would already exist, correct?

Reply:
@leahwicz, correct.  There's not a way I know of to only run if a specific label is added.  So this will run, then it hits a check for if the changelog exists:

https://github.com/dbt-labs/dbt-core/blob/efd1585966166c5e34735328eada291537163120/.github/workflows/dependency-changelog.yml#L61-L75

And should the pass on adding another one.

I could add another `if` to check if the action was `labeled` and the label was dependencies.  IT would still trigger the workflow but would fail sooner.  Something like this (syntax may be wrong):

```
jobs:
  dependency_changelog:
    if: |
      (${{ github.event == 'opened' }} && "contains(github.event.pull_request.labels.*.name, 'dependencies')")
      || (${{ github.event == 'labeled' }} && ${{ github.event.label.name == 'dependencies' }})
```

Reply:
Yeah what you have is fine then since it does check for the changelog existing so the danger of the workflow running multiple times is accounted for. 🚢 it!